tests/unit/jobserver/test_rap_api.py

```diff
@@ -157,12 +158,19 @@ def test_requests_post(self, rap_api_base_url, rap_api_token):
 class FakeResponse:
     """Lightweight fake response similar enough to requests.Response for these tests."""
 
-    def __init__(self, json_data, status_code=200):
-        self._json_data = json_data
+    def __init__(self, content=None, json_data=None, status_code=200):
+        if content and json_data:
+            raise ValueError("Tests should only specify content *or* json_data")
+
+        # Nb. in the real Response .content is a byte array, not a string
```

Comment:
Should we make it a byte array in tests too then?

Reply:
This would currently only really affect the error 500 codepaths, as all other tests interact with FakeResponse using only dicts - so it didn't seem to matter too much if it does dict->str->dict compared to dict->str->bytearray->str->dict for those tests.

With the current error 500 paths, I think it's currently resilient to whether it's a str or a bytearray (by which I mean it's sending it to a logger which I think will happily receive either), but maybe if it's worth doing it's worth doing accurately.

Reply:
done